environments/go/builder/build.sh

```diff
@@ -4,6 +4,16 @@ if [ -d ${SRC_PKG} ]
 then
     echo "Building in directory ${SRC_PKG}"
     cd ${SRC_PKG}
+
+    if [ -f glide.yaml ]; then
+        go get -u github.com/Masterminds/glide
+        glide install
+    fi
+
+    if [ -d vendor ]; then
+        cp -r vendor/* /usr/src/
```

Comment:
I'm a bit confused by this copying, why do we need this?

Reply:
Golang ignores the vendor folder for some reason. Since I needed it, I tried things and this worked ^^"

Might be because the usercode is not stored gopath (a friend suggested it ignores vendor then) or because we compile a plugin. Could not find a documented reason yet :/

Reply:
https://github.com/golang/go/issues/14566

Not having the usercode in gopath seems to be the actual problem.